spec/API_specification/array_object.md

```diff
@@ -375,20 +375,31 @@ Exports the array as a DLPack capsule, for consumption by {ref}`function-from_dl
 
 -   **stream**: _Optional\[int\]_
 
-    -   If given, the CUDA or ROCm stream number the consumer will use. Options are:
+    -   If given, the stream number the consumer will use on the device the array is present on, as a Python integer. Device-specific notes:
```

Comment:
Isn't the stream provided by the consumer and it's the producer's responsibility to add a wait / dependency on the provided stream if necessary?

Reply:
Yes. That's not in conflict with sentence right? The question is just whether or not to mention what you're saying here or in the DLPack docs. In this case it's one sentence, so probably good to add either here or in the _Semantics_ section under _Data interchange mechanisms_.

Reply:
Maybe this is just worded a bit awkwardly, where from reading it I got the impression the stream wasn't provided by the consumer.

Maybe we could word it a bit more explicitly to something like: "An optional pointer to a stream provided by the consumer that the producer will make the array safe to operate on"

Reply:
Thanks, adopted your wording.